bats/tests/snapshots/create-use-snapshot.bats

```diff
@@ -90,7 +92,14 @@ local_setup() {
     assert_success
     refute_output ""
     snapshot_id=$output
-    rdctl snapshot create "$snapshot_id"
+    snapshot_description="second snapshot made with the --description option with \\ and \" and '."
+    rdctl snapshot create "$snapshot_id" --description "$snapshot_description"
+    run rdctl snapshot list --json
+    assert_success
+    run jq_output "$(printf 'select(.name == "%s").description' "$snapshot_id")"
+    assert_success
+    assert_output --partial "$snapshot_description"
```

Comment:
Why is this a `--partial`? Shouldn't it be an exact match?

Reply:
There was a bug which might be fixed now.

Reply:
> There was a bug which might be fixed now.

If a test works around a bug that we expect to fix, then we need "BUG BUG BUG" comments to make that obvious